src/api/gitHub.js

```diff
@@ -111,32 +67,16 @@ export const fetchRepoInOrg = async (org, token, server, cursor) => {
  *
  * @param {string} org the org
  * @param {string} token the token
+ * @param {boolean} selfsigned the selfsigned option 
  * @returns {object} the fetched org information
  */
-export const fetchOrgInfo = async (org, server, token) => {
-  return await fetch(await determineGraphQLEndpoint(server), {
-    method: "POST",
-    headers: {
-      Authorization: `bearer ${token}`,
-    },
-    body: JSON.stringify({
-      query: `{
-        organization(login: "${org}") {
-          projects(first: 1) {
-            totalCount
-          }
-          membersWithRole(first: 1) {
-            totalCount
-          }
-        }
-      }`,
-    }),
-  })
+export const fetchOrgInfo = async (org, server, token, selfsigned) => {
+  return await fetch(await determineGraphQLEndpoint(server), await fetchOrgInfoOptions(org, token, selfsigned))
```

Comment:
The two parameters don't need to use `await`:

```suggestion
  return await fetch(determineGraphQLEndpoint(server), fetchOrgInfoOptions(org, token, selfsigned))
```

Reply:
I changed the logic as suggested.